sdk/types/src/types.input.ts

```diff
@@ -453,6 +453,10 @@ export interface RequestPictureOptions extends PictureOptions {
    * Flag that hints whether multiple cameras are being refreshed by this user request. Can be used to prefetch the snapshots.
    */
   bulkRequest?: boolean;
+  /**
+   * The maximum time in milliseconds to wait for the periodic picture to be taken.
+   */
+  periodicTimeout?: number;
```

Comment:
can you rename this to timeout, I'll use the property on all requests regardless of snapshot reason

Reply:
Fair enough, updated.